resolwe_bio/management/commands/generate_etc.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import csv
+import gzip
+import json
+import os
+import random
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.utils import timezone
+from resolwe.flow.models import Data, DescriptorSchema, Process, Storage
+from resolwe_bio.tools.utils import escape_mongokey, gzopen
+
+
+class Command(BaseCommand):
+
+    """Generate ETC objects"""
+
+    help = "Generate ETC objects"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-e', '--n-etc', type=int, default=3,
+                            help="Number of ETC objects to generate (default: %(default)s)")
+        parser.add_argument('--rseed', action='store_true', help="Use fixed random seed")
+
+
+    def set_user(self):
+        users = User.objects.filter(is_superuser=True).order_by('date_joined')
+        if not users.exists():
+            self.stderr.write("Admin does not exist: create a superuser")
+            exit(1)
+        return users.first()
+
+
+    @staticmethod
+    def create_etc(gene_ids, path, file_name):
+        genes = {}
+        with open(os.path.join(path, file_name), 'w') as f:
+            f.write('\t'.join(['Gene', '0', '4', '8', '12', '16', '20', '24']) + '\n')
+            with gzopen(gene_ids) as gene_ids:
+                all_genes = [line.strip() for line in gene_ids]
+                for gene in all_genes:
+                    expression = [str(random.gammavariate(1, 100)) for i in range(7)]
+                    expression = '\t'.join(expression)
+                    f.write('{}\t{}\n'.format(gene, expression))
+
+        with open(os.path.join(path, file_name)) as csvfile:
```

Comment:
Don't read newly created file. Generate data in memory and then write them in two files.